packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -202,7 +202,16 @@ const main = async () => {
         )
         .addOption(new program.Option('--noHMR', 'disable the client-side hot module replacement'))
         .action(async ({inspect, noHMR}) => {
-            execSync(`node${inspect ? ' --inspect' : ''} ${resolvedSSRPath}`, {
+            // We use @babel/node instead of node because we want to support ES6 import syntax
+            const babelNode = p.join(
+                require.resolve('webpack'),
```

Comment:
Why `webpack` rather than `babel-node` itself?

Reply:
because for some reason, babel packages have weird namings, i think babel node was called `babel-node` and since a major version (i forgot exactly which version), they moved to scoped packages like `@babel/node`.

However, `require.resolve` doesn't work for both `babel-node` nor `@babel/node`, so i had to fall back to webpack...

I also don't understand why all pwa-kit-dev commands resolves to bin using `path.join('..','..','..')`, but i just followed the existing pattern...

if anyone knows the reason, pls let me know.